src/test/kotlin/no/nav/familie/klage/behandling/BehandlingRepositoryTest.kt

```diff
@@ -45,7 +44,8 @@ class BehandlingRepositoryTest : OppslagSpringRunnerTest() {
         assertThat(behandling.id).isEqualTo(hentetBehandling.id)
         assertThat(behandling.fagsakId).isEqualTo(hentetBehandling.fagsakId)
         assertThat(behandling.eksternBehandlingId).isEqualTo(hentetBehandling.eksternBehandlingId)
-        assertThat(behandling.eksternFagsystemBehandlingId).isEqualTo(hentetBehandling.eksternFagsystemBehandlingId)
+        assertThat(behandling.påklagetVedtak.påklagetVedtakstype).isEqualTo(hentetBehandling.påklagetVedtak.påklagetVedtakstype)
+        assertThat(behandling.påklagetVedtak.eksternFagsystemBehandlingId).isEqualTo(hentetBehandling.påklagetVedtak.eksternFagsystemBehandlingId)
```

Comment:
Er ikke denne default null nå? Burde vi populere den for å teste mappingen? :)

Reply:
Ja, godt poeng :)